spec/services/checksum_validator_spec.rb

```diff
@@ -239,6 +239,11 @@
         expect(pres_copy.reload.status).to eq PreservedCopy::OK_STATUS
       end
 
+      it 'updates audit timestamps' do
+        expect(pres_copy).to receive(:update_audit_timestamps).with(true, true)
```

Comment:
i'd be inclined to query for the pres copy and check that the timestamps had the expected values (i.e. newer than when `cv.validate_checksums` got called), as opposed to checking that the update method got called.

Reply:
good suggestion, @jmartin-sul - check the desired outcome, not the methodology